src/main/java/lotto/domain/ticket/LottoTickets.java

```diff
@@ -33,6 +34,17 @@ private void validate(final List<LottoTicket> lottoTickets) {
                 .collect(Collectors.groupingBy(Function.identity(), Collectors.counting()));
     }
 
+    public int count() {
+        return lottoTickets.size();
+    }
+
+    public LottoTickets merge(final LottoTickets tickets) {
+        List<LottoTicket> mergeTickets = new ArrayList<>(lottoTickets);
+        mergeTickets.addAll(tickets.getLottoTickets());
```

Comment:
```suggestion
        mergeTickets.addAll(tickets.lottoTickets);
```
리마인드 차원에서! 요거 잊지 않으셨죠? ㅎㅎ